app/components/header.tsx

```diff
@@ -94,16 +98,36 @@ export default function Header() {
 
       <Flex justifyContent="flex-end" alignItems="center" color="white" gap="5" width="100%">
         <Hide below="lg">
-          <Text id="header-user">{user?.username}</Text>
+          <Text id="header-user">
+            {user.username === originalUser.username
+              ? user?.username
+              : `Impersonating: ${user?.username}`}
+          </Text>
         </Hide>
 
         <Menu>
           <MenuButton
             as={IconButton}
-            icon={<Avatar bg="brand.500" showBorder={true} borderColor="white" size="sm" />}
+            icon={
+              user.username !== originalUser.username ? (
+                <FaTheaterMasks size="35" />
```

Comment:
This doesn't seem to be working for me locally:

<img width="236" alt="Screenshot 2023-04-08 at 3 33 00 PM" src="https://user-images.githubusercontent.com/427398/230739517-15048040-69a7-4cf7-a039-9b8f01424701.png">

Reply:
I'm unsure why this is, which account did you log into to impersonate user3?

Reply:
I was logged in as `han.solo`.

Reply:
I can't seem to reproduce it

Reply:
Can you put up some screenshots?  It's likely something local for me.  If it works for you, we can ship to staging and test there.

Reply:
![image](https://user-images.githubusercontent.com/77400826/230740514-012a39de-37e0-4421-9ff1-be7ea5e9dfe0.png)
![image](https://user-images.githubusercontent.com/77400826/230740526-aff31d86-9493-4c1b-b666-103257c553ff.png)